### DISCUSSION TITLE
Why do some places on the curve have dots and some don't?

### DISCUSSION THREAD

**Discussion by CXX1113** [microsoft/nni#3748] [Q&A] [answered]
Only the data of the dot can be accessed by the mouse, but not by the dot. It makes me very confused.

**Describe the issue**:
Here is an example(TPE method)：
![5BEF809CE39B495B8DF2744E065C3DA9](https://user-images.githubusercontent.com/28216384/119337790-d00aa280-bcc1-11eb-8d78-779e2a194153.jpg)

**Environment**:
- NNI version: 0.2
- Training service (local|remote|pai|aml|etc): local
- Client OS: Ubuntu
- Server OS (for remote mode only): 
- Python version: 3.8
- PyTorch/TensorFlow version: 1.6.0
- Is conda/virtualenv/venv used?: yes
- Is running in Docker?: no

**Comment by kvartet** [comment, marked as answer]
To be honest, I don't fully understand what you mean. Are you saying that you can click on the small dots to access the detailed information of the trial, but you can’t click on other positions on the curve? Can you describe your problem again, of course, Chinese is okay. Maybe @Lijiaoa can offer you help. 

By the way, your NNI version is outdated. In the process of continuous evolution, NNI supports many new features. Perhaps you can upgrade NNI to v2.2.

**Comment by Lijiaoa** [comment]
@CXX1113 We connect these points to see the trend, so only the dots could be accessed by the mouse.
